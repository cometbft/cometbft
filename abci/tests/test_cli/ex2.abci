check_tx "abc"
check_tx "def=567"
finalize_block "def=567"
commit
finalize_block "hello=world"
commit
finalize_block "first=second"
commit
info
