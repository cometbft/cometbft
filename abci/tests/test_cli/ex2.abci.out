> check_tx "abc"
-> code: 2

> check_tx "def=567"
-> code: OK

> finalize_block "def=567"
-> code: OK
-> code: OK
-> data.hex: 0x0200000000000000

> commit 
-> code: OK

> finalize_block "hello=world"
-> code: OK
-> code: OK
-> data.hex: 0x0400000000000000

> commit 
-> code: OK

> finalize_block "first=second"
-> code: OK
-> code: OK
-> data.hex: 0x0600000000000000

> commit 
-> code: OK

> info 
-> code: OK
-> data: {"size":3}
-> data.hex: 0x7B2273697A65223A337D

